src/Peering/Peering/Microsoft.Azure.PowerShell.Cmdlets.Peering.Common.format.ps1xml

```diff
@@ -0,0 +1,1483 @@
+﻿<?xml version="1.0" encoding="utf-8" ?>
+<!--
+Code generated by Microsoft (R) PSSwagger 0.3.0
+Changes may cause incorrect behavior and will be lost if the code is regenerated.
+-->
+
+<Configuration>
+  <ViewDefinitions>
+    <View>
+      <Name>Microsoft.Azure.PowerShell.Cmdlets.Peering.Models.PSExchangeConnection</Name>
+      <ViewSelectedBy>
+        <TypeName>Microsoft.Azure.PowerShell.Cmdlets.Peering.Models.PSExchangeConnection</TypeName>
+      </ViewSelectedBy>
```

Comment:
Our default output format should be table instead of list. Please generate this using: https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/generate-format.ps1xml-file.md

Reply:
We looked at this as an option and we and our PMs agree that for the customer its important to be able to read the contents without having to use the $_.Field format. If this is a must, well do it, but as  you can see this is super custom formatting.